metagpt/actions/design_api.py

```diff
@@ -58,15 +59,15 @@
 """,
         "FORMAT_EXAMPLE": """
 [CONTENT]
-{
+{{
     "Implementation approach": "We will ...",
-    "project_name": "snake_game",
+    "project_name": "{project_name}",
```

Comment:
How about "Project name", keep the same.

Reply:
done